### PR TITLE
Timeline api endpoints

### PR DESCRIPTION
**Pull request by RMeissnerCC**
Adding stub endpoints to access timeline

### CODE REVIEW

File: src/app/api/api.py [hunk 1]
```diff
@@ -23,28 +23,62 @@
...
     responses={HTTP_404_NOT_FOUND: {"description": "Quality matrix not determinable"}},
-    tags=["Statistics"],
-    description="""Calculation of the quality matrix.
-    For each replication source and each property, e.g., `cm:creator`, the quality matrix returns the ratio of
-    elements which miss this entry compared to the total number of entries.
-    A missing entry may be `cm:creator = null`.""",
+    tags=[TAG_STATISTICS],
+    description=QUALITY_MATRIX_DESCRIPTION,
+)
+async def get_quality_matrix(timestamp: Optional[int] = None):
```
Comment (MRuecklCC) [inline src/app/api/api.py:41]: I assume this optional timestamp can be passed as query parameter (i.e. `.../quality_matrix?timestamp=XYZ`). If so, then we don't need the other endpoint. But instead add the query parameter documentation to the  docstring,

Comment (MRuecklCC) [reply src/app/api/api.py:41]: The first example here should also describe how to document a query parameter in the openapi spec!

https://fastapi.tiangolo.com/release-notes/?h=query#features

File: src/app/api/api.py [hunk 2]
```diff
@@ -23,28 +23,62 @@
...
+    "/quality_matrix/{timestamp}",
+    status_code=HTTP_200_OK,
+    response_model=List[ColumnOutputModel],
+    responses={HTTP_404_NOT_FOUND: {"description": "Quality matrix not determinable"}},
+    tags=[TAG_STATISTICS],
+    description=QUALITY_MATRIX_DESCRIPTION
+    + """A timestamp of the format XYZ yields the quality matrix at the respective date.""",
 )
-async def get_quality_matrix():
+async def get_past_quality_matrix(timestamp: Optional[int] = None):
```
Comment (MRuecklCC) [inline src/app/api/api.py:55]: not needed if timestamp is query parameter of above endpoint!

**Review by MRuecklCC — APPROVED** [review]
Not sure if we should merge it as is, or wait for the implementation to be more complete. I will leave that up to you :)

**Comment by RMeissnerCC** [comment]
> 

I will close this PR and add more functionality before the next review. I will see how I include your valuable points :)

**Pull request closed without merging**